addons/web/static/src/js/widgets/model_field_selector.js

```diff
@@ -226,7 +226,7 @@ var ModelFieldSelector = Widget.extend({
                         false,
                         ["store", "searchable", "type", "string", "relation", "selection", "related"]
                     ],
-                    context: this.getSession().user_context,
+                    context: Object.assign({field_selector: true}, this.getSession().user_context),
```

Comment:
why do we need to add field_selector in context here ?

Reply:
@dbeguin I don't think we need field_get. Could be handled with an option on `domain` widget. @pka-odoo would you do the changes ?